xknx/telegram/apci.py

```diff
@@ -643,9 +655,12 @@ class DeviceDescriptorRead(APCI):
 
     CODE = APCIService.DEVICE_DESCRIPTOR_READ
 
-    def __init__(self, descriptor: int = 0) -> None:
+    def __init__(self, descriptor: int = 0, is_numbered: bool = False) -> None:
         """Initialize a new instance of DeviceDescriptorRead."""
         self.descriptor = descriptor
+        self.additional_flags = None
+        if is_numbered:
```

Comment:
Why is is_numbered needed? For example `class Restart `uses `self.sequqence_number` in `to_knx()`

Reply:
Also I think it should be sequence instead of sequqence :)